execute/transformation.go

```diff
@@ -9,11 +9,17 @@ import (
 	"github.com/influxdata/flux/plan"
 )
 
+// Transformation represents functions that control of how the execution
+// engine will take in a flux Table, perform the necessary data processing
+// and return an output flux Table
```

Comment:
What about transformations that process more than one table?

Reply:
Good point. I can update the language to consider more than one table.

Reply:
The vocabulary we use is `stream` is a set of tables.

Reply:
@nathanielc I spoke with Chris a little bit about this and the Process comment so I made another change. Let me know what you think and I might mark it ready for review.